parm/config/config.resources.emc.dyn

```diff
@@ -338,7 +339,8 @@ elif [ $step = "eupd" ]; then
       export npe_eupd=42
       export nth_eupd=2
     fi
-    export npe_node_eupd=10
+    export npe_node_eupd=$(echo "$npe_node_max / $nth_eupd" | bc)
+    if [[ "$machine" = "WCOSS2" ]]; then export npe_node_eupd=9; fi
```

Comment:
This line is not necessary for WCOSS2.   128 / 14 = 9.142785... or 9 with integer math.   Similar comment applies to `config.resources.nco.static`

Reply:
@RussTreadon-NOAA You are correct, I have removed this line from both config.resource* files:
`if [[ "$machine" = "WCOSS2" ]]; then export npe_node_eupd=9; fi`
Generally, when the setup script math doesn't evenly divide (e.g. 128/14=9.1....) I tend to make sure the settings are forced so rocoto doesn't round up. For this situation though it rounds down so it's fine to not force it.